src/app/ui.js

```diff
@@ -35,6 +38,11 @@ class UI {
     });
   }
 
+  static reloadUI() {
+    const childs = dynamicDiv.querySelectorAll('.task-main');
+    childs.forEach((child) => child.remove());
```

Comment:
Kindly use `filter()` to remove your items.

- it's requested on your project requirements.
![Screen Shot 2021-12-04 at 8 03 13](https://user-images.githubusercontent.com/22258274/144712357-6b012058-2bab-4173-b1be-54d67611beba.png)

Reply:
Thank you for the review. But, once again the same. Let me explain again then. The requirement says "Clear all COMPLETED" that method you highlight removes EVERYTHING, not the completed ones!
Please check it once again or explain to me the need for filtering if I want to remove EVERYTHING even it's completed or not that method is removing ALL ITEMS and RESTORES. 
However, the clear all completed method is still there. Please kindly check the STORE class and the method 
![filter](https://user-images.githubusercontent.com/84543481/144712793-2c2643b5-3f27-48f8-87be-9bfc3a1ccf6b.PNG)

So the clear all completed button is USING the method deleteFromStore() not the reloadUI()! reloadUI is just reloads the DOM...